src/cclib/writer/ccwrite.py

```diff
@@ -28,7 +28,7 @@
 from . import xyzwriter
 
 
-def ccwrite(ccobj, outputtype=None, outputdest=None, returnstr=False,
+def ccwrite(ccobj, outputtype=None, outputdest=None, cjsonterse=False , returnstr=False,
```

Comment:
The 'cjsonterse' argument has been renamed to 'terse' and have gone ahead and included it in the function. Would be useful in the future